geojson_pydantic/features.py

```diff
@@ -16,10 +16,10 @@ class Feature(GenericModel, Generic[Geom, Props]):
     """Feature Model"""
 
     type: str = Field("Feature", const=True)
-    geometry: Geom
+    geometry: Geom = None
```

Comment:
```suggestion
    geometry: Optional[Geom] = None
```

Reply:
I agree that's easier to read. Do you think I should remove this line and incorporate the suggested change? https://github.com/developmentseed/geojson-pydantic/blob/54755500e5f5fce386b2be3e9cccb00dd830dbc1/geojson_pydantic/features.py#L12

Reply:
I opted for a via media: https://github.com/developmentseed/geojson-pydantic/pull/56/files#diff-a4fab2efe48317b94ba18803db374650b7685147f52df92c7998ad4176e4f675R12-R19

Reply:
Whoops my apologies I forgot that the `Optional` was rolled into the `TypeVar`, i'd prefer it to remain in the `TypeVar`.  My bad! 😬